### PR TITLE
Add searching cruft from home directory

### PR DESCRIPTION
**Pull request by JoosuaKoskinen**
## What are the main changes in this PR?

Adds some new known cruft files/dirs and
also searches for cruft in home directory.

##### Why are we doing this? Any context or related work?

Closes: #308.

#### Manual testing steps?

Create some files matching the cruft specifications
and let cruftfiles remover on security page take care of them.

### CODE REVIEW

File: lib/cruftfiles-ajax.php [hunk 1]
```diff
@@ -76,11 +82,15 @@ function seravo_ajax_list_cruft_files() {
...
         '*.wpress',
+        'core',
+        '*.bak',
       );
       // List of known cruft directories
       $list_dirs = array(
         'siirto',
-        'palautus',
+        '*palautus*',
+        '*restore*',
```
Comment (ottok) [inline lib/cruftfiles-ajax.php]: Some plugins or themes have files that include 'restore' in them for valid reasons. Our cruft remover should be as safe as possible. If we recommend removing too much customers might get angry when we "trick" them into deleting too much files from their site?

This is 99.9% correct it '*restore*' is found in the top level of a home directory, e.g. /home/user/yesterday-restore-backup, but it is only 90% correct if found somewhere else, e.g. /data/wordpress/htdocs/wp-content/plugins/xyz/lib/quick-restore/. Just saying we need to be conservative here, or add more constraints..

Comment (JoosuaKoskinen) [reply lib/cruftfiles-ajax.php]: True, we could make it so some are only searched from home directory.

Comment (JoosuaKoskinen) [reply lib/cruftfiles-ajax.php]: @ottok I decided to remove the 'restore' entry and added a warning message stating that not everything is safe to delete.

File: modules/security.php [hunk 2]
```diff
@@ -293,7 +293,7 @@ public static function logins_info_postbox() {
     public static function cruftfiles_postbox() {
       ?>
       <p>
-        <?php _e('Find and delete any extraneous and potentially harmful files taking up space in the file system.', 'seravo'); ?>
+        <?php _e('Find and delete any extraneous and potentially harmful files taking up space in the file system. Note that not everything is neccessarily safe to delete..', 'seravo'); ?>
```
Comment (elguitar) [inline modules/security.php]: Typo in necessarily

Comment (JoosuaKoskinen) [reply modules/security.php]: Fixed the typo

**Pull request merged**